release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -4000,4 +4000,41 @@ $ oc adm release info 4.10.59 --pullspecs
 [id="ocp-4-10-59-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-60"]
+=== RHBA-2023:3217 - {product-title} 4.10.60 bug fix and security update
+
+Issued: 2023-05-03
+
+{product-title} release 4.10.60, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2023:3217[RHBA-2023:3217] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3216[RHSA-2023:3216] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.60 --pullspecs
+----
+
+[id="ocp-4-10-60-features"]
+==== Features
+
+[id="ocp-4-10-60-control-for-verbosity-MetalLB-logs"]
+===== Controls for the verbosity of MetalLB logs
+
+* With this release, you can control the verbosity of MetalLB logs. You can control logging levels by using the following values for the `logLevel` specification in the MetalLB custom resource (CR):
+** all
+** debug
+** info
+** warn
+** error
+** none
+
+For example, you can specify the `debug` value to include diagnostic logging information that is helpful for troubleshooting.
```

Comment:
This and the next line aren't indented properly with the above bullet.

They should be aligned with the first bullet (not the subbullets). You can do this either by playing with the spacing (sometimes you can leave an empty line and then add the "+" to get it to indent properly. But the best way is usually to use `--` around the level 2 bullets, and pluses around everything else.